frontend/src/app/ui/components/bpm-input/bpm-input.component.html

```
@@ -1,5 +1,5 @@
-<div class="number-control">
+<form [formGroup]="form" class="number-control">
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)